server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -197,10 +213,83 @@ function markAppointmentAsReady($token, $firstName, $lastName, $emailAddress, $p
 	echo json_encode($response);
 }
 
+function storeConsent($reviewConsent, $virtualConsent, $signature) {
+	GLOBAL $DB_CONN;
+
+	$response = array();
+	$response['success'] = false;
+
+	try {
+		$DB_CONN->beginTransaction();
+
+		$virtualAppointmentConsentId = insertConsent($reviewConsent, $virtualConsent, $signature);
+		$DB_CONN->commit();
+
+		$response['success'] = true;
+		$response['virtualAppointmentConsentId'] = $virtualAppointmentConsentId;
+		// $response['message'] = AppointmentConfirmationUtilities::generateAppointmentConfirmation($appointmentId);
```

Comment:
We don't need this `response['message']`, do we? I think this was copied from signup, but we don't go to a confirmation page for uploading so I don't think this is needed at all.

Reply:
No--on the sign up page, the "message" is the appointment confirmation message that's sent to the FE. That shouldn't be relevant here